VERSIONING.md

```diff
@@ -0,0 +1,109 @@
+provide the library version of the implementation that is being compiled against. The function
+`const char* TBB_runtime_version();` can be used to query the library version of the oneTBB binary library that is
+loaded at runtime.
+
+## The oneTBB Interface Version
+
+The oneTBB interface version is the public contract about the interface to the library including the
+functions exported by the binary library and the types defined in the headers. The oneTBB library is designed
+for backwards compatibility. It works well in the scenarios where components compile/link against a TBB version
+X.Y. And then at runtime, the application loads a library version X.Z, where X.Z >= X.Y.
+
+A minor version may add new functionality in the headers or in the binary library, including new entry points to the
+binary library. An application compiled against an older minor version could not have used this new functionality
+and therefore can safely use the newer binary library.
+
+[As described in the specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onetbb/source/configuration/version_information),
+the macros `TBB_INTERFACE_VERSION_MAJOR`, `TBB_INTERFACE_VERSION_MINOR`, `TBB_INTERFACE_VERSION` in `oneapi/tbb/version.h`
+can be used to determine the interface version of the implementation that is being compiled against. The function
+`const char* TBB_runtime_interface_version();` can be used to query the interface version of the oneTBB binary library that is
+loaded at runtime.
+
+### Shared Library Practices for Linux
+
+On Linux we use a symlink system for binary libraries in the oneTBB distribution, and also explicitly set a soname.
+The actual shared library is libtbb.so.X.Y but it is built with a –soname that uses only the major version number,
+libtbb.so.X.  In addition, there is a symlink from libtbb.so.X -> libtbb.so.X.Y and a symlink from
+libtbb.so -> libtbb.so.X.  Therefore, whether you build against libtbb.so, libtbb.so.X or libtbb.so.X.Y, your
+application will have a dependency on libtbb.so.X.
+
+A common development process is to compile and link against a library version X.Y. This creates
+a dependency on libtbb.so.X. The packaged project then redistributes a TBB shared library that 
+is X.Z, where X.Z >= X.Y.
+
+In the (unusual) case where the package does not redistribute a TBB library and instead depends on a version already 
+available on the install platform, it is safest to build against the oldest version you expect to encounter; in the
+extreme this would be X.1. Of course, building against X.1 means no additional functionalility added in any later minor 
+release can be used during the development of the application.
+
+### Shared Library Practices for Windows
+
+On Windows, the TBB binary library has only the major version number, tbb12.dll.
+
+### How Backwards Compatibility is Maintained in the oneTBB library
```

Comment:
I believe this section corresponds to "4. *API versioning* used to maintain backwards compatibility" in the list above. Shouldn't the header be second-level (`##`) then?

Reply:
Fixed